README.md

```diff
@@ -136,40 +136,48 @@ directly with `poetry` rather than go through the package.
 brew install mysql postgresql # MacOS dependencies for C bindings
 poetry install
 ```
-
-**2. Download CSV of Testing Data**
+**2. Start Databases**
 
 ```shell-session
-wget https://files.grouplens.org/datasets/movielens/ml-25m.zip
-unzip ml-25m.zip -d dev/
+docker-compose up -d mysql postgres
 ```
 
-**3. Start Databases**
+**3. Run Unit Tests**
 
 ```shell-session
-docker-compose up -d mysql postgres
+poetry run python3 -m unittest
 ```
 
-**4. Run Unit Tests**
+**4. Seed the Database(s)**
+
+First, download the CSVs of seeding data:
 
 ```shell-session
-poetry run python3 -m unittest
-```
+wget https://files.grouplens.org/datasets/movielens/ml-25m.zip
```

Comment:
Link to 1m rows ;)

https://files.grouplens.org/datasets/movielens/ml-1m.zip

There's also 100k if you want it even faster. (see https://files.grouplens.org/datasets/movielens )